orchagent/bulker.h

```diff
@@ -455,6 +429,93 @@ class EntityBulker
     typename Ts::bulk_create_entry_fn                       create_entries;
     typename Ts::bulk_remove_entry_fn                       remove_entries;
     typename Ts::bulk_set_entry_attribute_fn                set_entries_attribute;
+
+    void flush_removing_entries(
+        _In_ std::vector<Te> rs)
+    {
+        size_t count = rs.size();
+        std::vector<sai_status_t> statuses(count);
+        sai_status_t status = (*remove_entries)((uint32_t)count, rs.data(), SAI_BULK_OP_ERROR_MODE_IGNORE_ERROR, statuses.data());
+        if (status == SAI_STATUS_SUCCESS)
```

Comment:
>status [](http://example.com/codeflow?start=12&length=6)

The status if useful, let's return it.

Reply:
Updated the functions to return the status.